src/datasource.ts

```diff
@@ -566,35 +570,44 @@ export class BigQueryDatasource {
       });
   }
 
-  private async doQueryRequest(query, requestId, maxRetries = 3) {
-    const path = `v2/projects/${this.runInProject}/queries`;
+  private async doQueryRequest(query, requestId, priority, maxRetries = 3) {
+    const location = this.queryModel.target.location || this.processingLocation || "US";
+    let data, queryiesOrJobs;
+    if(priority === 'INTERACTIVE'){
```

Comment:
Are you sure `priority` is always upper-case?

Reply:
It is coming from hard coded list, but it is a good practice to match the cases